theme/styles/index.sass

```diff
@@ -1,35 +1,12 @@
+@import 'common'
 
-// ## Crucial variables.
-// ## These aren't in _variables.sass so that functions can refer to them and subsequent variables can then be defined
-// ## in terms of those functions.
-$base-font-size: 16px // You REALLY don't want to change this.
-$w-px-epsilon: .5px
+@import '~@mangoweb/sass-base/animations/index'
+@import '~@mangoweb/sass-base/parts/shape'
 
-// ## Functions
-@import 'functions/*'
+@import '~bootstrap/scss/bootstrap-reboot'
 
-// ## Settings, variables and animations
-@import 'variables'
-@import 'animations'
-
-// ## Mixins
 @import 'mixins/*'
 
-// ## Frameworks
-@import '../../node_modules/normalize-scss/sass/normalize/import-now'
-
-// ## Web styles
-@import 'global'
-
-// ## Abstract
-@import 'abstract/*'
-
-// ## Components
-@import 'parts/*'
-
-// ## Compositions
-@import 'comps/*'
-
-// ## View components
-@import '../views/components/*.sass'
-
+@import 'parts/*.sass'
+@import '../views/components/**/*.sass'
+@import '../views/*.sass'
```

Comment:
Zavádíme tady najednou novou strukturu a zároveň mažeme komentáře v kódu. Prosím o osvětlení důvodů pro obé a plán, jak to sjednotit s `mango-cli-example` strukturou.